app/src/main/res/menu/home_menu.xml

```diff
@@ -50,6 +50,10 @@
       android:title="@string/menu_jump"
       app:showAsAction="never" />
   <item
+      android:id="@+id/translations"
+      android:title="Translations"
```

Comment:
thanks can fix that, there are items in the home_menu which should be converted as well